### PR TITLE
magento/merchdocs#: Customize the template. Add information how to add a custom CSS

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides information how to add a custom CSS to email templates. Also the proposed changes describes a nuance with usage:

```
{{template config_path="design/email/header_template"}}
```

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/email-template-message.html

## Links to Magento source code or PRs

- [{{var template_styles|raw}}](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Email/view/frontend/email/header.html#L23)

### CODE REVIEW

File: src/marketing/email-template-message.md [hunk 1]
```diff
@@ -52,7 +52,13 @@ The process of customizing the body of each message is the same as for customizi
...
    `customVar code=my_custom_variable`
 
-1. To make CSS declarations, enter the styles in **Template Styles**.
+1. To make CSS declarations, enter the styles in **Template Styles**.
+
+   ![Email templates - add custom styles]({% link images/images/email-template-add-custom-styles.png %}){: .zoom}
+   _Add Custom Styles_
+
+   {:.bs-callout-info}
+   Custom styles will be applied to email if `{{template config_path="design/email/header_template"}}` is present in the **Template Styles** otherwise does not. To use custom CSS without a default header template you need to provide them in the **Template Styles** in the `<style>` HTML tag.
```
Comment (leslietilling) [inline src/marketing/email-template-message.md:61]:
```suggestion
   Custom styles are applied to the email only if `{{template config_path="design/email/header_template"}}` is present in the **Template Styles** . To use custom CSS without a default header template, you must provide them here within the `<style>` HTML tag.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Some minor edits for clarity.

File: src/marketing/email-template-message.md (hunk 1 above)
Comment (atwixfirster) [reply src/marketing/email-template-message.md]: Hi, @leslietilling !

Your suggestion has been applied. 

Could you please review?

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Comment by leslietilling** [comment]
running tests

**Pull request merged**